blackbox/docs/release_notes/1.2.0.txt

```diff
@@ -0,0 +1,157 @@
+.. _version_1.2.0:
+
+=============
+Version 1.2.0
+=============
+
+Released on 2017/04/18.
+
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB
+   :ref:`version_1.1.2` or higher before you upgrade to 1.2.0.
+
+   You cannot perform a :ref:`cluster_upgrade` to this version. Any upgrade to
+   this version will require a full cluster restart.
+
+.. WARNING::
+
+   Before upgrading, you should `back up your data`_.
+
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+
+Breaking Changes
+================
+
+ - Due to the support of user defined functions there are new reserved keywords
```

Comment:
"To accommodate user-defined functions, some new reserved keywords have been added to the CrateBD dialect:"